dkg/messages.go

```diff
@@ -150,6 +158,55 @@ func (msg *Init) Decode(data []byte) error {
 	return json.Unmarshal(data, msg)
 }
 
+// Reshare triggers the resharing protocol
+type Reshare struct {
+	// ValidatorPK is the the public key to be reshared
+	ValidatorPK types.ValidatorPK
+	// EncryptedShares contains the encrypted shares of the old set
+	EncryptedShares map[types.OperatorID][]byte
+	// ThresholdOld is the threshold of the old set
+	ThresholdOld uint16
+	// OperatorIDs are the operators in the new set
+	OperatorIDs []types.OperatorID
+	// Threshold is the threshold of the new set
+	Threshold uint16
+}
+
+func (msg *Reshare) Validate() error {
+
+	if len(msg.ValidatorPK) != blsPubkeySize {
```

Comment:
why not make validatorpk byte[48] and that's it?

Reply:
It's an SSV original type we didn't attempt to change.